<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSettings</class>
 <widget class="QDialog" name="DlgSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/resources.qrc">
    <normaloff>:/hub/settings.png</normaloff>:/hub/settings.png</iconset>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_cancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btn_ok">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_common">
        <attribute name="title">
         <string>Common</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <widget class="QLabel" name="lbl_refresh_timer">
               <property name="text">
                <string>Refresh time (sec.) :</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="sb_refresh_timeout">
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="minimum">
                <number>10</number>
               </property>
               <property name="maximum">
                <number>600</number>
               </property>
               <property name="value">
                <number>120</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="lbl_notification_delay">
               <property name="text">
                <string>Notification delay (sec.) :</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="sb_notification_delay">
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="minimum">
                <number>3</number>
               </property>
               <property name="maximum">
                <number>300</number>
               </property>
               <property name="value">
                <number>5</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="lbl_logs_storage">
               <property name="text">
                <string>Logs storage:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="le_logs_storage"/>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="btn_logs_storage">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="lbl_ssh_keys_storage">
               <property name="text">
                <string>SSH-keys storage:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="le_ssh_keys_storage"/>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="btn_ssh_keys_storage">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="lbl_rtm_db_folder">
               <property name="text">
                <string>DB folder:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="le_rtm_db_folder"/>
             </item>
             <item row="2" column="2">
              <widget class="QPushButton" name="btn_rtm_db_folder">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="lbl_ssh_user">
               <property name="text">
                <string>EZ-SSH User:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="le_ssh_user"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_commands">
        <attribute name="title">
         <string>Commands</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QLineEdit" name="le_p2p_command"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lbl_ssh_command">
             <property name="text">
              <string>SSH command:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="lbl_p2p_path">
             <property name="text">
              <string>P2P command:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="le_ssh_command"/>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="btn_ssh_command">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="btn_p2p_file_dialog">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_rh_ip">
        <attribute name="title">
         <string>RH</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QGridLayout" name="gridLayout_8" columnstretch="1,3">
           <item row="0" column="0">
            <widget class="QLabel" name="lbl_rhip_host">
             <property name="text">
              <string>RH host:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="le_rhip_host"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lbl_rhip_port">
             <property name="text">
              <string>RH port:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="le_rhip_port"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lbl_rhip_user">
             <property name="text">
              <string>RH user:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="le_rhip_user"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="lbl_rhip_pass">
             <property name="text">
              <string>RH password:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="le_rhip_password">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_updating">
        <attribute name="title">
         <string>Updating</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QGridLayout" name="gridLayout_9" columnstretch="2,1,1">
           <item row="2" column="0">
            <widget class="QLabel" name="lbl_rh_update">
             <property name="text">
              <string>Resource Host check for updates frequency:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lbl_tray_update">
             <property name="text">
              <string>SubutaiTray check for updates frequency:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="cb_tray_frequency"/>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="cb_rh_frequency"/>
           </item>
           <item row="1" column="2">
            <widget class="QCheckBox" name="chk_tray_autoupdate">
             <property name="text">
              <string>Automatic tray update</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QCheckBox" name="chk_rh_autoupdate">
             <property name="text">
              <string>Automatic rh update</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="lbl_p2p_update">
             <property name="text">
              <string>P2P check for updates frequency:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="cb_p2p_frequency"/>
           </item>
           <item row="0" column="2">
            <widget class="QCheckBox" name="chk_p2p_autoupdate">
             <property name="text">
              <string>Automatic p2p update</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="lbl_rhm_update">
             <property name="text">
              <string>Management check for updates frequency:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="cb_rhm_frequency"/>
           </item>
           <item row="3" column="2">
            <widget class="QCheckBox" name="chk_rhm_autoupdate">
             <property name="text">
              <string>Automatic mh update</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
